modules/node-shared/src/main/scala/org/tessellation/node/shared/http/routes/SigningMessageRoutes.scala

```diff
@@ -0,0 +1,40 @@
+package org.tessellation.node.shared.http.routes
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.option._
+
+import org.tessellation.node.shared.infrastructure.snapshot.services.SigningMessageService
+import org.tessellation.node.shared.infrastructure.snapshot.storage.SigningMessageStorage
+import org.tessellation.routes.internal._
+import org.tessellation.security.{Hasher, SecurityProvider}
+
+import eu.timepit.refined.auto._
+import org.http4s.HttpRoutes
+import org.http4s.dsl.Http4sDsl
+
+// TODO proper location/module for this class?
```

Comment:
Currency L0 is sufficient as far as I'm concerned, considering this is a very lightly used event I don't think we need to make another layer of indirection in the APIs in terms of storing on blocks, but open to discussion on that.